wayland-protocols/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
     - `xdg-dialog-v1`
     - `linux-drm-syncobj-v1`
 
+### Breaking changes
+- `set_constraint_adjustment`/`SetConstraintAdjustment` now takes a `ConstraintAdjustment` instead of a u32.
```

Comment:
LGTM, but I try to keep the "breaking changes" section at the top of the changelog for each releases in general